src/components/Header/DesktopHeader.tsx

```diff
@@ -0,0 +1,98 @@
+import { useEffect, useRef, useState } from 'react';
+
+import LanguageSelector from './LanguageSelector';
+import Logo from './Logo';
+import SearchInput from './SearchInput';
+import Sublist from './Sublist';
+
+import { AboutListData, StoreListData } from '../../constants/header';
+
+const DesktopHeader: React.FC = () => {
+  const [desktopMenuOpen, setDesktopMenuOpen] = useState<boolean>(false);
+  const [curMenu, setCurMenu] = useState<number>(1);
```

Comment:
u can implement it more clearly like this  const [curMenu, setCurMenu] = useState<"store" | "about">("store");